InnerEye/ML/metrics_dict.py

```diff
@@ -824,3 +827,6 @@ def flush(self, log_info: bool = False) -> None:
         if log_info:
             s = df.to_string(index=False, float_format="%.6f")
             logging.info(f"\n{s}")
+
+
+MetricsPerExecutionModeAndEpoch = Dict[ModelExecutionMode, Dict[Union[int, str], ScalarMetricsDict]]
```

Comment:
perhaps better defined at the top of the file?

Reply:
nope cause we need ScalarMetricsDict to be defined first

Reply:
but you can use a forward type declaration as a string: `'ScalarMetricsDict'`

Reply:
Updated :)